### DISCUSSION TITLE
Let's earn the Galaxy Brain achievement

### DISCUSSION THREAD

**Discussion by omersahin3** [community/community#18645] [Other Feature Feedback, Questions, & Ideas] [answered]
<html>
<body>

Galaxy Brain | Bronze 🥉  --> 8 accepted answers

Galaxy Brain | Silver 🥈  --> 16 accepted answers

Galaxy Brain | Gold 🥇 --> 32 accepted answers

</body>
</html>

**Comment by mnislam01** [comment]
Well... Here is my first answer. If you accept it then I am closer.

**Reply by jazzybruno** [reply]
Strange but this is my comment too

**Reply by RahulRaj352001** [reply]
Strange but this is my comment too

**Reply by venus01015** [reply]
well, all right

**Reply by Wep-wep** [reply]
Lograste tu objetivo?

**Reply by Sayrix** [reply]
Strange but this is my comment too

**Reply by Anmol-Baranwal** [reply]
Is this achievement worth showing?

- One can learn how this achievement works but the actual purpose is that you have actually helped someone in the community, and as a reward you have earned this badge, right?

**Reply by thiag0-dot** [reply]
ok

**Comment by mnislam01** [comment]
And this is my second answer.

**Reply by number0x0001** [reply]
lol you can't mark two comments as answers

**Reply by mnislam01** [reply]
Good to know. Expecting points for the try.

**Reply by thiag0-dot** [reply]
ok

**Reply by abenkoula71** [reply]
ok

**Reply by shubheshswain91** [reply]
ok

**Comment by sam0hack** [comment]
Great but does this works like that?

**Reply by radar-blip** [reply]
No.

**Comment by Gary-H9** [comment]
Disappointing to see people spamming this.

**Reply by omersahin3** [reply]
we're just trying to learn how to win

**Reply by VerzatileDev** [reply]
I agree, this is horrible that peopple are spamming it, and ruining it for everyone else that is interested in helping the community. Just for the achivement.

**Comment by Tran-Quyen** [comment]
It is very helpfull for me. Thank for your question

**Comment by tanny99** [comment]
Hi this is my answer for your problem!

**Reply by sourabmaity** [reply]
https://github.com/Schweinepriester/github-profile-achievements/discussions/104
check this

**Reply by Upchein** [reply]
I don't really understand it

**Comment by tanny99** [comment]
Please accept my answers!!

**Comment by tanny99** [comment]
Please accept my answer sir!

**Comment by RahulRaj352001** [comment]
Let's earn the Galaxy Brain achievement
#18645

this can be a new Github society

**Comment by foxyseta** [comment]
Here is my answer!

**Comment by BinaryDex** [comment]
here i am!

**Reply by RahulRaj352001** [reply]
okay

**Comment by Vault11Overseer** [comment]
Please accept my answer good sir.

**Comment by LucasJohnNyamhanga** [comment]
Am i late?

**Reply by Wep-wep** [reply]
Estás a tiempo

**Comment by yuwangi** [comment]
here i am!

**Reply by ThingkingInCoffee** [reply]
where are you

**Reply by shloke** [reply]
come out to eat bbq

**Reply by yuwangi** [reply]
when?

**Comment by Vault11Overseer** [comment]
I don't understand how to get the Galaxy Brain achievement.
Can someone please assist?

**Reply by RahulRaj352001** [reply]
I don't understand how to get the Galaxy Brain achievement.
Can someone please assist?

**Comment by Ahmed-Jedidi** [comment]
@AlteredArt to achieve the 'Galaxy Brain' achievement, take note of the following:

- Answer at least 2 discussions.
- - Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
- - Discussions should be in a public repository.
- - There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

Hope this answer has helped you, and all the best in getting the achievement! :)

**Reply by rvsfara** [reply]
Do I need to answer about any activity?

**Comment by w4rf0t** [comment]


**Reply by RahulRaj352001** [reply]
Hello... Here is my first answer. Hope you will accept my answer, so I am closer.

**Comment by msuliq** [comment]
Only one answer can be marked in any one given question, there is no point in replying here since the top answer has already received the green checkmark.

**Reply by RahulRaj352001** [reply]
Only one answer can be marked in any one given question, there is no point in replying here since the top answer has already received the green checkmark.

**Comment by abenkoula71** [comment]
hello

**Reply by DatDerpyDude81** [reply]
Hi

**Reply by RahulRaj352001** [reply]
hello

**Reply by abenkoula71** [reply]
how are you

**Comment by RahulRaj352001** [comment]
hello anyone can answer how to take this

**Comment by amansingh14** [comment]
Thanks accept my answer

**Comment by ashcala** [comment]
I'm here

**Comment by FriendlyUser** [comment]
I'm here for the galaxy brain :D

**Comment by cx0y** [comment]
Let's go!

**Reply by AMAZZAL-AMINE** [reply]
never giveup

**Comment by thiag0-dot** [comment]
I'm here for the galaxy brain :D

**Reply by AMAZZAL-AMINE** [reply]
you chauld be a great programer

**Comment by thiag0-dot** [comment]
I'm here

**Reply by AMAZZAL-AMINE** [reply]
good luck

**Reply by abenkoula71** [reply]
okey

**Comment by AMAZZAL-AMINE** [comment]
anser me guys

**Comment by rvsfara** [comment]
Do I need to answer about any activity?

**Reply by Kris-fynn** [reply]
yes

**Comment by bryandbernhardt** [comment]
Ok, help me 🥲

**Comment by osmancitci** [comment]
How will I do..

**Comment by SaifMahmud366** [comment]
i' newbie here.. please mark my comment as answer😊

**Reply by cx0y** [reply]
ok 😁

**Reply by SaifMahmud366** [reply]
OMG...did you watch anime?
drop your myanimelist account here😁

**Reply by cx0y** [reply]
yes ! 😁 <https://myanimelist.net/profile/cx0y>

**Reply by SaifMahmud366** [reply]
i messaged you there😁😁

**Comment by brunoalksouza** [comment]
Please mark my answer

**Comment by ParanoidUser** [comment]
**For anyone interested:** We run a simple [galaxy trivia](https://github.com/ParanoidUser/yolo/discussions/categories/galaxy-trivia) game, where people try to answer the questions to earn *Galaxy Brain* badges. New questions are posted each morning and the winners are determined the next day. You can also find [helpful guides](https://github.com/ParanoidUser/yolo/discussions/categories/hunt-log) for other types of badges, with screenshots showing exactly what you need to do to earn them. Our community is growing, if you are willing to contribute, leave a message to [subscribe](https://github.com/ParanoidUser/yolo/discussions/categories/subscribe).

**Comment by github-staff** [comment, marked as answer]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Reply by VerzatileDev** [reply]
Support the decision!

**Reply by fraineralex** [reply]
I think it's an excellent decision.

**Comment by TheRealHex** [comment]
**Mark** me too lol :)

**Comment by ironcg20** [comment]
Obtaining <Galaxy brain>  can be helpful for top-rated search results in Git Hub or google search?

**Comment by Hyadawild** [comment]
so to get this achievement get a little hard to do but with little effort we can get this achievement, it was quite interesting and became a bit more challenging

**Comment by RahulRaj352001** [comment]
Your galaxy trivia game sounds like a fun and engaging way to learn more about galaxies and space! It's great that you're offering guides with helpful screenshots for earning different types of badges. Your growing community will likely enjoy the trivia challenges and the opportunity to earn badges. If someone wants to join your game or subscribe to your community, they can leave a message to subscribe. Let me know if there's anything else I can do to assist you!

**Comment by RahulRaj352001** [comment]
It sounds like you've made a necessary decision to maintain the quality and integrity of your GitHub Community by disabling the ability to earn Achievements there. It's unfortunate that users were creating low-quality, nonsensical, or spam posts to earn Achievements, but your action is likely to result in a better experience for the rest of the community.

Thank you for sharing the information and the link to the announcement post. It's helpful for those affected to understand the reasoning behind the change and the scope of its impact. If you have any further questions or concerns, feel free to ask.

**Comment by danialtavakoli** [comment]
My answer to get this badge. Thank you.

**Comment by andrijajovanovic98** [comment]
thanks :D
